cmd/lncli/main.go

```diff
@@ -349,6 +372,14 @@ func main() {
 				"macaroon jar instead of the default one. " +
 				"Can only be used if profiles are defined.",
 		},
+		cli.StringSliceFlag{
+			Name: "metadata",
```

Comment:
We need to remember that `With great power comes great responsibility` :+1:

Reply:
😎 💪

Reply:
do you think i should add some kind of warning?

Reply:
I guess contributors and reviewers won't allow to just put extra stuff there instead of using actual fields when needed. I think it's good as it is :+1: